pycontw/pycon.tw
# Add labels field to AbstractProposal and present in the talk list page

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies


## Description
**Describe what the change is**
Add a generic `labels` field to `AbstractProposal`, and present the "Community Label" in the Talk List page.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to Django Admin and edit any accepted TalkProposal
2. In the labels field, put in `Community Track`
3. Go to the Talk List page, and see the result
4. Click on the `Community Track` label

## Expected behavior
There should be a `Community Track` or `社群軌` label next to the title of the Talk. After clicking the label, it will jump to the bottom, and you should see a short description of the Community Track.

**Additional context**
This is a generic field, but maybe not ideal. I was thinking that maybe in the future, the field could hold multiple labels, achieved by making them a comma separated string.

But currently, the label "Community Track" is hard coded in the Talk List page, and also in the translation.
